CMakeLists.txt

```diff
@@ -123,8 +124,11 @@ set(SWIFT_INSTALL_COMPONENTS "${_SWIFT_KNOWN_INSTALL_COMPONENTS}" CACHE STRING
 # * toolchain-dev-tools -- install development tools useful in a shared toolchain
 # * dev -- headers and libraries required to use Swift compiler as a library.
 
-set(SWIFT_SDKS "" CACHE STRING
-    "If non-empty, limits building target binaries only to specified SDKs (despite other SDKs being available)")
+set(SWIFT_CROSS_COMPILE_STDLIB_TARGETS "" CACHE STRING
+    "A list of deployment-targets to cross-compile the standard-library for (by default, the standard library is always compiled for the host)")
+
+set(SWIFT_SKIP_HOST_STDLIB "" CACHE STRING
+  "If True, will not build the standard library for the host")
```

Comment:
Is this note still true? There's not much harm in adding a CMake option that doesn't quite work right now, but I think a better approach would be to only add ones that already work as intended.

Would it be possible to either:

1. Fix the issues you describe first, before merging this pull request?
2. Or add this flag in a later pull request? It seems to be used in five locations, so maybe it's too much trouble.